app/src/main/res/values/styles.xml

```diff
@@ -274,7 +274,8 @@
     <item name="backgroundTint">@color/selector_color_primary</item>
     <item name="android:clickable">true</item>
     <item name="android:foreground">@drawable/white_circle_bound_touch_feedback</item>
-    <item name="android:src">@drawable/ic_favorite_empty</item>
+    <item name="android:src">@drawable/animselector_ic_favorite</item>
+    <item name="android:scaleType">center</item>
```

Comment:
Any reason why we are including `scaleType` here? Should be set by default on a FAB

Reply:
Yup. I'm overriding it since the `VectorDrawable`s I provided have extra padding inside the viewport. I'm setting the `scaleType` to `center` to make sure the icon is occupying `24dp` when at rest and is properly centered (otherwise it would be really tiny).